pkg/controller/migplan/migplan_controller.go

```diff
@@ -289,13 +289,11 @@ func (r *ReconcileMigPlan) Reconcile(request reconcile.Request) (reconcile.Resul
 		}
 
 		if migAnalytic != nil {
-			// Process PV Capacity
-			r.processExtendedPVCapacity(plan, migAnalytic)
-			// raise condition for unconfirmed PV sizes
-			r.validatePVSizeConfirmation(plan, migAnalytic)
```

Comment:
I guess we don't need to validate PV size confirmation anymore?

Reply:
@djwhatle correct. we only need to set warnings for 1.4.1